3-platform/order-ms/app/pom.xml

```
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.6core.platform</groupId>
+        <artifactId>order-ms</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>app</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.6core.platform</groupId>
+            <artifactId>domain</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```

bad name for groupId and artifactId. 
We have a bunch of services, and with such naming we have a lot of overlapping dependency (you are expecting here is order-related domain, someone else will expect the warehouse related domain).
Group Id describes company name, so you can left as it is. 
Change Artifact Id. 
```suggestion
            <groupId>com.6core.platform</groupId>
            <artifactId>domain</artifactId>
            <version>0.0.1-SNAPSHOT</version>
```